lib/Net/DRI/DRD/GMORegistry/JPRS.pm

```diff
@@ -0,0 +1,133 @@
+## Domain Registry Interface, JPRS GMO Registry (.jp) Driver
+##
+## Copyright (c) 2021 Patrick Mevzek <netdri@dotandco.com>. All rights reserved.
+##           (c) 2021 Michael Holloway <michael@thedarkwinter.com>. All rights reserved.
+##           (c) 2021 Paulo Jorge <paullojorgge@gmail.com>. All rights reserved.
+##
+## This file is part of Net::DRI
+##
+## Net::DRI is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+##
+## See the LICENSE file that comes with this distribution for more details.
+####################################################################################################
+
+package Net::DRI::DRD::GMORegistry::JPRS;
+
+use strict;
+use warnings;
+
+use base qw/Net::DRI::DRD/;
+
+use DateTime::Duration;
+use Net::DRI::Util;
+
+__PACKAGE__->make_exception_for_unavailable_operations(qw/contact_transfer contact_transfer_start contact_transfer_stop contact_transfer_query contact_transfer_accept contact_transfer_refuse domain_renew/);
+
+=pod
+
+=head1 NAME
+
+Net::DRI::DRD::GMORegistry::JPRS - JPRS GMO Registry (.JP) Driver for Net::DRI
+
+=head1 DESCRIPTION
+
+Additional domain extensions for JPRS GMO Registry (.JP) ccTLDs
+
+JP is operated by JPRS GMO Registry
+
+Terms
+
+- JPRS : Japan Registry Service
+
+- o-JP : Organizational Type JP Domain Name (co.jp, or.jp, ed.jp, ac.jp)
+
+- g-JP : General-use JP Domain name (.jp)
+
+This DRD extends the L<Net::DRI::DRD::GMORegistry::JPRS>
+
+=head2 Standard extensions:
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::secDNS> urn:ietf:params:xml:ns:secDNS-1.1
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::GracePeriod> urn:ietf:params:xml:ns:rgp-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::LaunchPhase> urn:ietf:params:xml:ns:launch-1.0
```

Comment:
Does it use LaunchPhase? It's in this file and Extensions::JP but I don't think its used anywhere, and probably not required...?

Reply:
Not described in their documentation (as many other things that are in use but are not listed as well) but te extension is listed in the greeting. Why I've added.

Reply:
Ok thanks for clarifying